docs/csharp/language-reference/operators/index.md

```diff
@@ -10,11 +11,31 @@ helpviewer_keywords:
   - "expressions [C#]"
 ms.assetid: 0301e31f-22ad-49af-ac3c-d5eae7f0ac43
 ---
-# C# operators (C# reference)
+# C# operators and expressions (C# reference)
 
-C# provides a number of operators supported by the built-in types. For example, [arithmetic operators](arithmetic-operators.md) perform arithmetic operations with numeric operands and [Boolean logical operators](boolean-logical-operators.md) perform logical operations with the [bool](../builtin-types/bool.md) operands. Certain operators can be [overloaded](operator-overloading.md). With operator overloading, you can specify the operator behavior for the operands of a user-defined type.
+C# provides a number of operators. Many of them are supported by the [built-in types](../builtin-types/built-in-types.md) and allow you to perform basic operations with values of those types. For example, [arithmetic operators](arithmetic-operators.md) perform arithmetic operations with numeric operands and [Boolean logical operators](boolean-logical-operators.md) perform logical operations with [`bool`](../builtin-types/bool.md) operands. Others can be used in a broader scope. For example, you can use [conditional operator `?:`](conditional-operator.md) or [`switch` expression](switch-expression.md) to select which operand expression to evaluate based on a Boolean condition or a matched pattern, respectively.
```

Comment:
The last two sentences seem unnecessary and are hard to follow without examples.  I would consider omitting them.

On the other hand, I think the mention of operator overloading is important here and shouldn't be dropped.